tree: tag RESET statements as RESET

otan opened the pull request:
Release note (sql change): We now correctly send the RESET tag instead
of the SET tag when a RESET statement is run.

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/69053)
<!-- Reviewable:end -->

otan commented:
bors r=richardjcai

otan commented:
oops

bors r-

otan committed:
tree: tag RESET statements as RESET

Release note (sql change): We now correctly send the RESET tag instead
of the SET tag when a RESET statement is run.

otan commented:
bors caught a rebase conflict! whoohoo!

otan commented:
bors r=richardjcai
